experiments/ClimaEarth/components/atmosphere/climaatmos.jl

```diff
@@ -210,6 +236,11 @@ function Interfacer.update_field!(atmos_sim::ClimaAtmosSimulation, ::Val{:co2},
     end
 end
 # extensions required by the Interfacer
+function Interfacer.update_field!(sim::ClimaAtmosSimulation, ::Val{:emissivity}, field)
+    # sets all 16 bands (rows) to the same values
+    sim.integrator.p.radiation.rrtmgp_model.surface_emissivity .=
+        reshape(CC.Fields.field2array(field), 1, length(parent(field)))
+end
```

Comment:
In the long term, maybe we can make the RRTMGPInterface assume every band is the same if only one value is passed in (maybe this is already the behavior not? I'm not sure). This way we don't need to hard code the length(parent) here.